client/api_group.go

```diff
@@ -176,7 +211,21 @@ func (c *Client) HeadGroupMember(ctx context.Context, groupName string, groupOwn
 	return err == nil
 }
 
-// PutGroupPolicy apply group policy to user specified by principalAddr, the sender need to be the owner of the group
+// PutGroupPolicy apply group policy to user specified by principalAddr, the sender needs to be the owner of the group
```

Comment:
Is this true for the owner must be the group owner? If the owner grants permission to other accounts, can that account do this operation?

Reply:
Can we add more details about principalAddr, and what is the policy format and scope?

Reply:
only resources owner can put policy.

Reply:
ok